src/pages/Embeds/Plio.vue

```diff
@@ -55,6 +55,7 @@
           :videoPlayerElementId="plioVideoPlayerElementId"
           v-model:isFullscreen="isFullscreen"
           v-model:responseList="itemResponses"
+          :configuration="plioSettings.player.children.configuration"
```

Comment:
I will move it to a computed variable but there's no need to check for an invalid value.
Pliosettings will never be null because in case it's null, it'll inherit from global default settings.
Plus where I'm sending it, to the ItemModal, that already has a fallback value to work on.
So basically this check is happening over there.